js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -404,17 +410,20 @@ function propagateSelfHeal () {
   } catch (e) {
     console.errorStack(e, 'Failure during propagate self heal');
   }
-  reloadRouting();
```

Comment:
Having this hidden in here made troubleshooting a bit more difficult than necessary, so I'd rather have it explicit every time.